### ISSUE TITLE
Repeating I/O instructions can panic

### ISSUE THREAD

**Issue by omarandlorraine** [ivanizag/rust-z80#7]
I'm getting this error

```
thread 'main' panicked at [...]/iz80-0.4.0/src/opcode_io.rs:107:36:
attempt to subtract with overflow
```

Looking at that line of the source, it's a `+` and `-`, which should be a `.wrapping_add` or `.wrapping_sub`.

What do you think? Want me to submit a PR?

**Comment by ivanizag** [comment]
Yes, that is problem. The subtraction is the problem as the value is an u8 extended to u16. Having wrapping for both is the way to go.

A PR would be welcome.

**Comment by ivanizag** [comment]
Thanks for the contribution. I published v0.4.1 with the fix.
